src/LCT.PackageIdentifier/BomCreator.cs

```diff
@@ -40,8 +40,8 @@ public async Task GenerateBom(CommonAppSettings appSettings, IBomHelper bomHelpe
 
             // Calls package parser
             listOfComponentsToBom = await CallPackageParser(appSettings);
-            Logger.Logger.Log(null, Level.Notice, $"No of components to BOM after removing bundled, dev dependency & " +
-                $"internal components = {listOfComponentsToBom.Components.Count}", null);
+            Logger.Logger.Log(null, Level.Notice, $"No of components to BOM after removing bundled & exluded components " +
+                $"= {listOfComponentsToBom.Components.Count}", null);
```

Comment:
Rephrase as "No of components added to BOM after removing bundled & excluded components"